draft-ietf-dnsop-svcb-https.md

```diff
@@ -1177,6 +1173,9 @@ the client SHOULD construct a corresponding "https" URL as follows:
 
 This construction is equivalent to {{Section 8.3 of HSTS}}, point 5.
 
+If the "http" origin exists and contains important resources that have no
```

Comment:
I feel like "important" may be a sticking point here.  I infer what you mean, but there's no definition of "important" that someone could point to confirm that they are compliant with this MUST.  Is it sufficient to simply say "contains resources"?  The content of a 301 isn't a resource, it's merely the payload of a redirect.

Reply:
Here's another variation:

>If this redirection would result in a loss of functionality (e.g. important
resources that are only available on the "http" origin), the operator MUST
NOT publish an HTTPS RR.